astropy/utils/data.py

```diff
@@ -44,9 +42,13 @@ class Conf(_config.ConfigNamespace):
     Configuration parameters for `astropy.utils.data`.
     """
 
+    # NOTE: Make sure all the dataurl values have trailing "/".
```

Comment:
Maybe put this in the "help" string (on line 48) instead?  While unlikely, random user *might* decide to re-assign the data site to some local copy and they should know the trailing ``/`` is necessary.